diff --git a/dist-cjs/chains/EVM/helpers.js b/dist-cjs/chains/EVM/helpers.js
index cc3b76a440a6a182ba84cbe689962702a31a02c1..05fe4abb0372b902aacbf0ea185bb25dd4e4fee2 100644
--- a/dist-cjs/chains/EVM/helpers.js
+++ b/dist-cjs/chains/EVM/helpers.js
@@ -2,7 +2,7 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.toHex = exports.getSubstrateRecipientAddressInBytes = exports.getEVMRecipientAddressInBytes = exports.constructSubstrateRecipient = exports.createPermissionlessGenericDepositData = exports.createPermissionedGenericDepositData = exports.createERCDepositData = void 0;
 const ethers_1 = require("ethers");
-const create_1 = require("@polkadot/types/create");
+const create_1 = require("../../../node_modules/@polkadot/api/node_modules/@polkadot/types/create");
 const util_crypto_1 = require("@polkadot/util-crypto");
 const registry = new create_1.TypeRegistry();
 /**
diff --git a/dist-esm/chains/EVM/helpers.js b/dist-esm/chains/EVM/helpers.js
index caee2324bdb05ec255c82659aa96788f022e593f..47af8dfb4759e9b2566580dcff4157a9fba53b20 100644
--- a/dist-esm/chains/EVM/helpers.js
+++ b/dist-esm/chains/EVM/helpers.js
@@ -1,5 +1,5 @@
 import { utils, BigNumber } from 'ethers';
-import { TypeRegistry } from '@polkadot/types/create';
+import { TypeRegistry } from '../../../node_modules/@polkadot/api/node_modules/@polkadot/types/create';
 import { decodeAddress } from '@polkadot/util-crypto';
 const registry = new TypeRegistry();
 /**
